spec/controllers/api/popolo/motions_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+describe API::Popolo::MotionsController do
+
+  let(:public_group) { create :group, is_visible_to_public: true }
+  let(:private_group) { create :group, is_visible_to_public: false }
+
+  let(:public_discussion) { create :discussion, group: public_group, private: false }
+  let(:private_discussion) { create :discussion, group: private_group }
+
+  let(:public_motion) { create :motion, discussion: public_discussion }
+  let(:private_motion) { create :motion, discussion: private_discussion }
+
+  before { public_motion; private_motion }
+
+  describe 'index' do
```

Comment:
Yay it does all the filtering / paging nonsense already without me doing anything!